### ISSUE TITLE
Cats Effect: Schedule

### ISSUE THREAD

**Issue by jdegoes** [zio/interop-cats#23]
Introduce an effect-polymorphic class atop `ZSchedule`, with a default implementation for any Cats Effect data type (Cats IO, Monix Task, etc.).

```scala
package scalaz.zio.interop

trait Schedule[F[_], -A, +B] {
  ...
}
object Schedule {
  def never[F[_]: ConcurrentEffect]: Schedule[F, Any, Nothing] = ???
}
```

The methods will all delegate to `ZSchedule`, but they will take the `ZIO` effect returned by `ZSchedule` methods and lift it into any effect `F[_]` that is suitable equipped.

**Comment by GodPlaysChess** [comment]
I thought this issue is done and was merged.
Is there still anything to do with this issue?

**Comment by ghostdogpr** [comment]
Closed by #2 

@GodPlaysChess you're right, thanks!
